spring-web/src/main/java/org/springframework/web/client/DefaultResponseErrorHandler.java

```diff
@@ -175,11 +207,29 @@ private String getErrorMessage(
 	 * @see HttpServerErrorException#create
 	 */
 	protected void handleError(ClientHttpResponse response, HttpStatus statusCode) throws IOException {
+		handleError(null, null, response, statusCode);
+	}
```

Comment:
This is a protected method that is now bypassed. Something to consider for a minor/major release but not at this stage of the 5.3.x branch.

Reply:
Thanks for looking into that!

Initially, I didn't think of squeezing it into 5.3. I'd be very happy to do so - but I think the previous changes broke something in spring-integration, so it might not be that easy.

I tried to not to change the behaviour of this method at all. I can also try to rewrite this to make it more 5.3-compatible. But If you think it makes more sense to wait - let's wait.